webrtc-stats.html

```diff
@@ -1470,13 +1470,13 @@ <h2>
                 <p>
                   There are various reasons why the jitter buffer delay might be increased to a higher value, such as
                   to achieve AV synchronization or because a
-                  <a href="https://w3c.github.io/webrtc-extensions/#dom-rtcrtpreceiver-playoutdelay">playoutDelay</a>
+                  <a href="https://w3c.github.io/webrtc-extensions/#dom-rtcrtpreceiver-jitterbuffertarget">jitterbuffertarget</a>
```

Comment:
https://w3c.github.io/webrtc-extensions/#dom-rtcrtpreceiver-jitterbuffertarget
```suggestion
                  <a href="https://w3c.github.io/webrtc-extensions/#dom-rtcrtpreceiver-jitterbuffertarget">jitterBufferTarget</a>
```